packages/project-access/src/translations/project-access.i18n.json

```diff
@@ -0,0 +1,6 @@
+{
+    "error": {
+        "projectRootNotFound": "Could not find any project root for {{-path}}. Parameter 'sapuxRequired' was {{sapuxRequired}}",
+        "moduleNotInstalled": "Module {{moduleName}} not installed in project {{-projectRoot}}"
```

Comment:
Similar to the other translatable string: is this a message that is ever intended to be shown in a user interface? If it is just for logging, then not translation is required and if it is to be handled by the consumer, maybe a designated Error type would be a better option.

Reply:
same as above, we need a decision on whether text in OUTPUT need translation. But keep in mind the previously decision was to translate.

Reply:
Discussed at https://github.com/SAP/open-ux-tools/pull/685#discussion_r975145025.
Note: the text was not translated in the Fiori tools, it was ready for translation.